dist/schema.json

```diff
@@ -231,11 +231,6 @@
                   "items": {
                     "$ref": "#/components/schemas/Account"
                   }
-                },
-                "examples": {
-                  "Account200Example": {
```

Comment:
Keep the `Account200Example` references for responses that return `#/components/schemas/Account`